config/deps/deps.exs

```diff
@@ -1,5 +1,9 @@
 import Config
 
+config :ex_aws,
+  access_key_id: [{:awscli, "default", 30000}],
+  secret_access_key: [{:awscli, "default", 30000}]
```

Comment:
This assumes that the configuration being used is the default configuration. This won't work for people for whom these credentials are under another configuration name.

Reply:
Good call, I'll switch it to using the system's via (from `ExAws`):

> Alternatively, if you already have a profile name set in the AWS_PROFILE environment variable, you can use that with `{:awscli, :system, timeout}`